projects/plugins/jetpack/extensions/blocks/ai-assistant/ai-assistant.php

```diff
@@ -189,3 +189,17 @@ function () {
 		}
 	}
 );
+
+/**
+ * Register the `ai-seo-enhancer` extension.
+ */
+add_action(
+	'jetpack_register_gutenberg_extensions',
+	function () {
+		if ( apply_filters( 'jetpack_ai_enabled', true ) &&
+			apply_filters( 'ai_seo_enhancer_enabled', false )
+		) {
+			\Jetpack_Gutenberg::set_extension_available( 'ai-seo-enhancer' );
+		}
+	}
+);
```

Comment:
We need to clean up this file, there are some features that can be removed. But for this PR we can just add this to the umbrella `if` up on line 115, as `breve_enabled` is defined.

Reply:
Good catch! I rarely visit this file, didn't notice there was a more tidy section for this. Thanks!